db/config.cc

```diff
@@ -155,6 +155,12 @@ struct convert<db::config::seed_provider_type> {
 
 }
 
+#if defined(CONFIG_ENABLE_SSTABLE_KEY_VALIDATION) || defined(DEBUG)
```

Comment:
What is this `CONFIG_ENABLE_SSTABLE_KEY_VALIDATION`? Is this a way to force validation at build time?